temporal-serviceclient/src/main/java/io/temporal/serviceclient/ChannelManager.java

```diff
@@ -39,7 +39,13 @@
 final class ChannelManager {
   private static final Logger log = LoggerFactory.getLogger(ChannelManager.class);
 
-  private static final int MAX_INBOUND_MESSAGE_SIZE = 25_000_000;
+  /**
+   * This value sets the limit on the incoming responses from Temporal Server to Java SDK.
+   * It doesn't affect the limit that Temporal Server exposes on the messages coming to
+   * Temporal Frontend from Java SDK.
+   * The Server Frontend limit on incoming gRPC message value is currently 4Mb.
+   */
+  private static final int MAX_INBOUND_MESSAGE_SIZE = 128 * 1024 * 1024;
```

Comment:
What about outbound? Did we decide on max size for that?

Reply:
The server does it by limiting its inbound. And it's 4Mb now (gRPC default)